request_clinical_results_for_uf_affiliates.R

```diff
@@ -0,0 +1,84 @@
+library(tidyverse)
+library(dotenv)
+library(REDCapR)
+library(lubridate)
+library(sendmailR)
+
+source("R/data_fetching.R")
+source("R/data_writing.R")
+
+# set the timezone
+script_name <- "request_clinical_results_for_uf_affiliates"
+script_run_time <- set_script_runtime()
+Sys.getenv("INSTANCE")
+
+read_contacts_data <- read_redcap_db(
+    redcap_uri = Sys.getenv("URI"),
+    token = Sys.getenv("CONTACT_TOKEN"),
+    project_title = Sys.getenv("CONTACT_PROJECT_TITLE"),
+    filter_logic = "[contact_clinical_results___1] = '1'",
```

Comment:
This logic gets you the result I think you wanted

```suggestion
    filter_logic = "[contact_tracing_arm_1][contact_clinical_results(1)] = '1'",
```